README.md

```diff
@@ -32,8 +32,11 @@ We want to create website with Django, that will be useful to as many as three d
 5. Install [pre-commit](https://pre-commit.com/)
 6. In terminal go to repository folder and enter and `pre-commit install`
 7. Open IDE (preferably PyCharm) and with `Open folder` option open repository folder as project in your IDE
+8. Install [Docker Desktop](https://www.docker.com/products/docker-desktop/)
```

Comment:
Docker Desktop isn't important. Write something like: "Install docker compose (on windows whole docker desktop)" idk

Reply:
I think "Install docker" is enough
```suggestion
8. Install [Docker](https://www.docker.com/)
```